src/main/java/io/stargate/sgv2/jsonapi/service/shredding/Shredder.java

```diff
@@ -363,12 +363,12 @@ private void validateArrayValue(String referringPropertyName, JsonNode arrayValu
         if (DocumentConstants.Fields.VECTOR_EMBEDDING_FIELD.equals(referringPropertyName)) {
           if (arrayValue.size() > limits.maxVectorEmbeddingLength()) {
             throw ErrorCode.SHRED_DOC_LIMIT_VIOLATION.toApiException(
-                "number of elements Vector embedding ('%s') has (%d) exceeds maximum allowed (%d)",
+                "number of elements Vector embedding (field '%s') has (%d) exceeds maximum allowed (%d)",
                 referringPropertyName, arrayValue.size(), limits.maxVectorEmbeddingLength());
           }
         } else {
           throw ErrorCode.SHRED_DOC_LIMIT_VIOLATION.toApiException(
-              "number of elements an indexable Array ('%s') has (%d) exceeds maximum allowed (%d)",
+              "number of elements an indexable Array (field '%s') has (%d) exceeds maximum allowed (%d)",
```

Comment:
Was missing this from Array and Object constraints failures: added to make messages more uniform.